applications/tune_psf.py

```diff
@@ -208,61 +208,61 @@ def addParameters(
         mrra2 = np.random.uniform(max(mrra2_0 - mrra2_range, 0), mrra2_0 + mrra2_range)
         mar = np.random.uniform(max(mar_0 - mar_range, 0), mar_0 + mar_range)
-        addParameters(mrra, mar, mrf, mrra2)
+        add_parameters(mrra, mar, mrf, mrra2)
 
     # Loading measured cumulative PSF
-    dataToPlot = OrderedDict()
+    data_to_plot = OrderedDict()
     if args_dict["data"] is not None:
-        dataFile = gen.findFile(args_dict["data"], args_dict["model_path"])
-        dataToPlot["measured"] = loadData(dataFile)
-        radius = dataToPlot["measured"]["Radius [cm]"]
+        data_file = gen.find_file(args_dict["data"], args_dict["model_path"])
+        data_to_plot["measured"] = load_data(data_file)
+        radius = data_to_plot["measured"]["Radius [cm]"]
 
     # Preparing figure name
-    plotFileName = "_".join((label, telModel.name + ".pdf"))
-    plotFile = outputDir.joinpath(plotFileName)
-    pdfPages = PdfPages(plotFile)
+    plot_file_name = "_".join((label, tel_model.name + ".pdf"))
+    plot_file = output_dir.joinpath(plot_file_name)
+    pdf_pages = PdfPages(plot_file)
 
-    def calculateRMSD(data, sim):
+    def calculate_RMSD(data, sim):
         """
         Calculates the Root Mean Squared Deviation to be used
         as metric to find the best parameters.
         """
         return np.sqrt(np.mean((data - sim) ** 2))
 
-    def runPars(pars, plot=True):
+    def run_pars(pars, plot=True):
         """
         Runs the tuning for one set of parameters, add a plot to the pdfPages
         (if plot=True) and returns the RMSD and the D80.
         """
-        telModel.changeMultipleParameters(**pars)
-
-        ray = RayTracing.fromKwargs(
-            telescopeModel=telModel,
-            simtelSourcePath=args_dict["simtelpath"],
-            sourceDistance=args_dict["src_distance"] * u.km,
-            zenithAngle=args_dict["zenith"] * u.deg,
-            offAxisAngle=[0.0 * u.deg],
+        tel_model.change_multiple_parameters(**pars)
+
+        ray = RayTracing.from_kwargs(
+            telescope_model=tel_model,
+            simtel_source_path=args_dict["simtelpath"],
+            source_distance=args_dict["src_distance"] * u.km,
+            zenith_angle=args_dict["zenith"] * u.deg,
+            off_axis_angle=[0.0 * u.deg],
         )
 
         ray.simulate(test=args_dict["test"], force=True)
-        ray.analyze(force=True, useRX=False)
+        ray.analyze(force=True, use_rx=False)
 
         # Plotting cumulative PSF
         im = ray.images()[0]
-        d80 = im.getPSF()
+        d80 = im.get_psf()
 
         # Simulated cumulative PSF
-        dataToPlot["simulated"] = im.getCumulativeData(radius * u.cm)
+        data_to_plot["simulated"] = im.get_cumulative_data(radius * u.cm)
 
-        rmsd = calculateRMSD(
-            dataToPlot["measured"]["Cumulative PSF"], dataToPlot["simulated"]["Cumulative PSF"]
+        rmsd = calculate_RMSD(
```

Comment:
to be consistent, I think we should use lowercase for RMSD

Reply:
Done!